org.nelson.Nelson.yaml

```diff
@@ -0,0 +1,314 @@
+app-id: org.nelson.Nelson
```

Comment:
`nelson.org` is parked domain for sale. 

But you checked:
> The domain used for the application ID is [controlled by the application developer(s)](https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository) and the [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

So something is incorrect here.

Reply:
it will be: io.github.nelson-lang.Nelson in this case ??

Reply:
'io.github.nelson-lang.Nelson' is not a valid application name: Only last name segment can contain -

Reply:
so, what is the good id with good domain ?
'io.github.nelson.Nelson' ?

Reply:
'io.github.nelson_lang.Nelson'